src/main/java/underdogs/devbie/answer/domain/Answer.java

```diff
@@ -0,0 +1,51 @@
+package underdogs.devbie.answer.domain;
+
+import java.util.Objects;
+
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import underdogs.devbie.config.BaseTimeEntity;
+import underdogs.devbie.exception.CreateFailException;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+public class Answer extends BaseTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private Long userId;
+
+    private Long questionId;
+
+    @Embedded
+    private AnswerContent content;
+
+    @Builder
+    public Answer(Long userId, Long questionId, AnswerContent content) {
+        validateParameters(userId, questionId, content);
+        this.userId = userId;
+        this.questionId = questionId;
+        this.content = content;
+    }
+
+    private void validateParameters(Long userId, Long questionid, AnswerContent content) {
+        if (Objects.isNull(userId) | Objects.isNull(questionid) | Objects.isNull(content)) {
```

Comment:
비트연산자(`|`) 보다는 논리연산자(`||`)를 사용하는게 좋을 것 같습니다.
논리연산자(`||`)는 여러개의 조건 중 하나라도 만족하지 않으면 뒤에 조건은 무시하고 결과를 리턴하기 때문입니다.

```suggestion
        if (Objects.isNull(userId) || Objects.isNull(questionid) || Objects.isNull(content)) {
```

- 참고링크
[|| operator in Java - GeeksforGeeks](https://www.geeksforgeeks.org/operator-in-java/)

Reply:
감사합니다 반대로 생각해서 저렇게 했네요! :) 

[링크](https://stackoverflow.com/questions/4014535/differences-in-boolean-operators-vs-and-vs)